tests/common/test_logger.py

```diff
@@ -192,3 +194,18 @@ def test_telemetry_logger1(self, mock_save):
 
             elif x['name'] == 'Context3':
                 self.assertEqual(x['value'], '')
+
+    def test_logger_should_log_in_utc(self):
+        file_name = "test.log"
+        file_path = os.path.join(self.tmp_dir, file_name)
+        test_logger = Logger()
+        test_logger.add_appender(logger.AppenderType.FILE, logger.LogLevel.INFO, path=file_path)
+
+        before_write_utc = datetime.utcnow().strftime(u'%Y/%m/%d %H:%M:%S')
+        test_logger.info("The time should be in UTC")
+
+        with open(file_path, "r") as log_file:
+            log = log_file.read()
+            time_in_file = log.split(LogLevel.STRINGS[logger.LogLevel.INFO])[0]
+
+            self.assertIn(before_write_utc, time_in_file)   # Comparing the UTC timestamp till seconds only
```

Comment:
but what if the minute flips? xx:10:59 to xx:11:00?

Reply:
oh, i misunderstood your comment - the second may change too, right?

Reply:
I considered that scenario (it has a higher probability of second flipping too), but I feel we shouldn't see this problem as it would all happen within ms. 
Another way to approach this test is to mock utcnow() to return some random time and then verify if the time matches. I didn't want to go that route as I wanted to test the scenario with correct data. Do you feel that would be a better test?

Reply:
> we shouldn't see this problem 

 famous last words :) 

maybe you can take the delta between the two times and assert that it is within a few sec (different time zone would be hours)

Reply:
Good point. I'll add the delta

Reply:
Done